### PR TITLE
Batchnorm Variant#0 ASM changes

### PR DESCRIPTION
**Pull request by muralinr**
Batchnorm Variant#0 ASM changes

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -29,36 +29,39 @@ cmake_policy(SET CMP0057 NEW)
...
+#This is incremented when the ABI to the library changes
 set( MIOpen_SOVERSION 1.0 )
 
 
 function(add_kernels KERNEL_FILES)
     set(INIT_KERNELS_LIST)
     foreach(KERNEL_FILE ${KERNEL_FILES})
         if("${CMAKE_VERSION}" VERSION_LESS 3.0)
-            configure_file(${KERNEL_FILE} ${KERNEL_FILE}.delete)
+            configure_file(${
```
Comment (daniellowell) [inline src/CMakeLists.txt:58]: Looks like you formatted your CMakeLists.txt
Please revert the formatting to all CMake files as it can introduce bugs.

Comment (muralinr) [reply src/CMakeLists.txt:58]: Sure Daniel.

**Comment by daniellowell** [comment]
@muralinr ASM code issues.
Also, can you post some relative performance numbers?

**Comment by muralinr** [comment]
> @muralinr ASM code issues.
> Also, can you post some relative performance numbers?

Hi Daniel, I am looking issues.  I will post numbers once I address code issues.

**Pull request closed without merging**